airflow/models/dagrun.py

```diff
@@ -135,7 +135,7 @@ def find(dag_id=None, run_id=None, execution_date=None,
         :param run_id: defines the run id for this dag run
         :type run_id: str
         :param execution_date: the execution date
-        :type execution_date: datetime.datetime
+        :type execution_date: datetime.datetime, list
```

Comment:
Please explain this.

Reply:
We can pass execution_date as a list of dates. I added list to allowed types. Did i miss something ?

Reply:
```suggestion
        :type execution_date: datetime.datetime or list[datetime.datetime]
```

Then

Reply:
Done